mc2skos/mc2skos.py

```diff
@@ -284,8 +283,8 @@ def main():
 
     if args.infile is None:
         raise ValueError('Filename not specified')
-
-    in_file = args.infile
+    else:
```

Comment:
If the condition above is met, an exception is raised, so `else` is not necessary here.

Reply:
ok, changed